README.md

```diff
@@ -195,6 +204,44 @@ I, [2023-11-23T22:41:39.148523 #1]  DEBUG -- : [my-own-prefix] Request has been
 I, [2023-11-23T22:41:39.148537 #1]  DEBUG -- : [my-own-prefix] Returning the response from the original request.
 ```
 
+### manage_redis_exceptions
+
+By default, the `grape-idempotency` gem is configured to handle potential `Redis` exceptions.
+
+However, this approach carries a certain level of risk. In the case that `Redis` experiences an outage, the idempotent functionality will be lost, and this issue may go unnoticed.
+
+If you prefer to take control of handling potential `Redis` exceptions, you have the option to configure the gem to abstain from managing Redis exceptions.
+
+```ruby
+Grape::Idempotency.configure do |c|
+  c.storage = @storage
+  c.manage_redis_exceptions = false
+end
+```
+
+Now, if a `Redis` exception arises while attempting to utilize the `Redis` storage, the gem will re-raise the identical exception to your application. Thus, you will be responsible for handling it within your own code, such as:
```

Comment:
Should we make it clear that it could be that it fails when setting the idempotency key before executing the action or after? So it might be possible that it fails after the action is executed? 
Just thinking on cases where it fails after,  and you don't want to return a 500 as the action really has been completed succesfully but what failed is storing the response in the idempotency storage

Reply:
Thanks for the comment @salrepe ! 

Considering that scenario, it means that the idempotency key was stored and it exists in the DB (because it was stored at the beginning of the process). But when trying to store the response, the redis raises an exception.

If we manage the exception, we will return the response.

However, if the consumer retries the same operation with the same idempotency-key, we will not be able to return a proper response, as it was not properly stored in the database.

Any suggestion for how should we behave in that case?

Reply:
Change applied :) now, if an exception appears AFTER the wrapped code was performed, nothing is re-raised. README updated for explaining this behavior.